### PR TITLE
Added validation for Create Post endpoint

### PR DESCRIPTION
**Pull request by en-aleksandar-stojmenovic**

### CODE REVIEW

File: ProjectSetup/appsettings.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "UIEndpoint": "v1/swagger.json"
   },
   "JwtSettings": {
-    "Secret": ""
+    "Secret": "JWTAuthenticationHIGHsecuredPasswordVVVp1OH7Xzyr"
```
Comment (bronjaharis) [inline ProjectSetup/appsettings.json]: Please leave this empty.

File: ProjectSetup/Controllers/V1/PostController.cs [hunk 2]
```diff
@@ -45,8 +45,14 @@ public async Task<ActionResult<Post>> Get([FromRoute] Guid postId)
 		[Authorize(Roles = UserRoles.Admin)]
 		[HttpPost(ApiRoutes.Post.Create)]
 		[CustomExceptionFilter(typeof(CategoryBadRequestException), HttpStatusCode.BadRequest)]
+		[CustomExceptionFilter(typeof(PostBadRequestException), HttpStatusCode.BadRequest)]
 		public async Task<IActionResult> Create([FromBody] CreatePostRequest postRequest)
 		{
+			if (!ModelState.IsValid)
```
Comment (bronjaharis) [inline ProjectSetup/Controllers/V1/PostController.cs]: Please remove this whole if statement. You don't need this.

File: ProjectSetup/appsettings.json (hunk 1 above)
Comment (en-aleksandar-stojmenovic) [reply ProjectSetup/appsettings.json]: I've fixed it.

File: ProjectSetup/Controllers/V1/PostController.cs (hunk 2 above)
Comment (en-aleksandar-stojmenovic) [reply ProjectSetup/Controllers/V1/PostController.cs]: I've removed it.

**Review by bronjaharis — APPROVED** [review]

**Pull request merged**